server/models/models.go

```diff
@@ -17,6 +20,13 @@ type Project struct {
 	Evolutions       []Evolution
 }
 
+// Validate checks if the model is valid
+func (project Project) Validate() error {
```

Comment:
This should not be in the models package.

The validate call as well as the model (for project and account) should be moved to their respective packages. This will require a find and replace of `models.Account` to `accounts.Account` and `models.Project` to `projects.Project`

Reply:
Validate() is a method, It has to be on the same package with its struct.
Why do you need this has to be moved?

Reply:
The models package shouldn't exist.

The correct way to have the models would be to have them in their proper packages. When I originally created the project I put everything into models incorrectly and now they need to be moved out as they are worked on. Since the validation should be in the accounts and projects packages respectively, they need to be moved.

As more work is done, the other models will be moved to their respective packages as well.

Reply:
since Project extends Evolution struct, and Evolution extends Task, I think it is too complicated If I move Project into its package, and Account as well